configure.ac

```diff
@@ -444,18 +444,28 @@ AC_ARG_ENABLE(ncp-app,
             enable_ncp_app=${enableval}
             enable_ncp_app_mtd=${enableval}
             enable_ncp_app_ftd=${enableval}
+            enable_ncp_app_radio=${enableval}
```

Comment:
With `--ncp-app=radio`, a new app only has radio layer will be built. With `--enable-raw-link-api`, *raw link api* will be enabled in `ftd` and `mtd` mode app. And `--ncp-app=radio` implicitly enables *raw link api*.